web/app/components/workflow/run/status-container.tsx

```diff
@@ -18,30 +18,30 @@ const StatusContainer: FC<Props> = ({
     <div
```

Comment:
the svgs from `/app/components/workflow/run/assets` where not copied to the assets folder